nat-lab/bin/cleanup_natlab_processes

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo "Executing natlab process cleanup script"
+
+for pid in $(ps -e -o pid=); do
+    # Skip non-testing processes
+    if ! grep --null-data --text KILL_ID /proc/${pid}/environ; then
```

Comment:
Test if `pid` exist. 
Kill command may take some time - so that here is race condition (starting from second loop iteration), grep may drop here 'no search file or directory'.

Entire loop can't be all the time safe - but lets reduce possibility of race conditions

Reply:
good point, I see two races here:
* process exits while loop is running, which would make `kill` fail. I missed this case, as it would exit the loop in such scenario. I will fix it.
* The other is that PIDs, at least theoretically, may be reused, if they wrap around. This one, IMHO, is pretty hard to deal with without introducing overkill solutions, and I am considering to leave it unsolved.

Reply:
updated